src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Filters } from '../../types/Filters';
+
+type Props = {
+  filterBy: string,
+  setFilterBy: (val: Filters) => void,
+};
+
+export const Filter: React.FC<Props> = ({
+  filterBy,
+  setFilterBy,
+}) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        data-cy="FilterLinkAll"
+        href="#/"
+        className={
+          filterBy === 'all'
+            ? 'filter__link selected'
+            : 'filter__link'
+        }
```

Comment:
use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally